Cargo.toml

```diff
@@ -13,7 +13,7 @@ Library to support the reading and writing of zip files.
 edition = "2018"
 
 [dependencies]
-flate2 = { version = "1.0", default-features = false, optional = true }
+flate2 = { version = ">=1.0.0, <=1.0.14", default-features = false, optional = true }
```

Comment:
Can we add a comment to the issue noting why we have such a strange version requirement?